src/components/flatlist/FlastListNewGoodsHome.tsx

```diff
@@ -0,0 +1,199 @@
+import {
+  View,
+  Text,
+  FlatList,
+  Image,
+  TouchableOpacity,
+  StyleSheet,
+} from 'react-native';
+import React, {useEffect, useState} from 'react';
+import color from '../../assets/colors/color';
+import LinearGradient from 'react-native-linear-gradient';
+import {useNavigation} from '@react-navigation/native';
+import {HomeScreenNavigationProp} from '../../Screens/tabBarScreen/HomeScreen';
+import {DETAIL_GOOD} from '../../assets/source/navigate';
+import {SHOP_CART} from '../../assets/source/image';
+import {useAppDispatch, useAppSelector} from '../../redux/hooks';
+import {HStack} from 'native-base';
+import {
+  starImgFilled,
+  startImgCorner,
+} from '../../Screens/cartScreen/RatingScreen';
+import {axiosAddProduct} from '../../Screens/cartScreen/reduxCart/CartSlide';
+
+const FlastListNewGoodsHome = () => {
+  const navigation = useNavigation<HomeScreenNavigationProp>();
+
+  const topSellHome = useAppSelector(state => state.home.dataTopSell);
+  const dataNewSell = topSellHome?.listNewProduct.items;
+
+  const dispatch = useAppDispatch();
+
+  const handleAddCart = (item: any) => {
+    const product = {id: item.id};
+
+    dispatch(axiosAddProduct(product));
+  };
+
+  const RenderItem =
+    (navigation: HomeScreenNavigationProp) =>
+    ({item}: any) => {
+      const goDetailGood = () => {
+        navigation.navigate(DETAIL_GOOD, {item: item});
+      };
+      const totalRate = [1, 2, 3, 4, 5, 6];
+      const rate: number = item.ratingAvg;
+      return (
+        <TouchableOpacity onPress={goDetailGood}>
+          <View style={styles.container}>
+            {item.image !== null && (
+              <Image
+                source={{uri: item.image[0]}}
+                style={styles.imageWrapper}
+              />
+            )}
+            <View style={styles.wrapper}>
+              <Text style={styles.textName}>{item.name}</Text>
+              {item.packingSize !== null && (
+                <Text style={styles.textAmount}>
+                  {item.packingSize.height}x{item.packingSize.width}x
+                  {item.packingSize.length} (cm)
+                </Text>
+              )}
+              <HStack>
+                {item.ratingAvg !== null ? (
```

Comment:
check lưu ý trong MS Team thread
```suggestion
                {!!item.ratingAvg ? (
```

Reply:
fix done